discovery_engine_core/core/src/engine.rs

```diff
@@ -48,15 +48,21 @@ pub enum Error {
     /// Invalid stack id: {0}.
     InvalidStackId(StackId),
 
-    /// An operation on a stack failed: {0}.
+    /// An operation on a stack failed with a stack error: {0}.
```

Comment:
Maybe we call it internal operation on stack failed? so we don't have to specify the kind of the error in the string.

Reply:
I removed `StackOpGen` so this string has been reverted back